.github/workflows/manual.yaml

```diff
@@ -0,0 +1,47 @@
+name: Build Without Cache
+
+on: workflow_dispatch
+    # can maybe specify PR branch ref here?
+    # https://docs.github.com/en/actions/reference/events-that-trigger-workflows#example-workflow-configuration
+
+env:
+  DOCKER_IMAGE: uwhackweek/snowex:latest
```

Comment:
we'll want to replace any occurance of 'snowex' to 'template' otherwise the docker image could get overwritten